docs/user/02-configuration.md

```diff
@@ -73,6 +73,24 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
 | **specHash** (required)                              | integer    |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
-| **state** (required)                                 | string     | Can have one of the following values: Ready, Error, Processing, Warning. Ready state is set when all the resources are deployed successfully and backend is connected. It gets Warning state in case backend is not specified and NATS module is not installed or EventMesh secret is missing in the cluster. Error state is set when there is an error. Processing state is set if recources are being created or changed. |
+| **state** (required)                                 | string     | Can have one of the following values: `Ready`, `Error`, `Processing`, `Warning`. `Ready` state is set when all the resources are deployed successfully and backend is connected. It gets `Warning` state in case backend is not specified, NATS module is not installed or EventMesh secret is missing in the cluster. `Error` state is set when there is an error. `Processing` state is set if recources are being created or changed. See the next section for more details. |
 
 <!-- TABLE-END -->
+
+<h3>More Details About Eventing CR State</h3>
+The following table provides more details on the overall state of the Eventing CR:
+- The <b>State</b> column has the overall state of the Eventing CR.
+- The <b>Backend Config</b> column is part of the eventing CR that is either available or mistakenly not specified by a user.
+- The <b>Backend State</b> column describes the state of the NATS backend or EventMesh Secret existence. 
+
+Warnings indicate that user action is required, that is, the user must install the NATS module or create an EventMesh Secret.
+
+| Backend Config | Backend State         | Backend Type | Eventing State | Addition comment |
```

Comment:
Imho it would be easier to grep if we would sort the table by impact on event flow.